compiler/src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -45,17 +45,26 @@ import reporting.diagnostic.messages.SuperCallsNotAllowedInline
  *  (11) Minimizes `call` fields of `Inline` nodes to just point to the toplevel
  *       class from which code was inlined.
  *
+ *  (12) Converts GADT bounds into normal type bounds
+ *
  *  The reason for making this a macro transform is that some functions (in particular
  *  super and protected accessors and instantiation checks) are naturally top-down and
  *  don't lend themselves to the bottom-up approach of a mini phase. The other two functions
  *  (forwarding param accessors and synthetic methods) only apply to templates and fit
  *  mini-phase or subfunction of a macro phase equally well. But taken by themselves
  *  they do not warrant their own group of miniphases before pickling.
  */
-class PostTyper extends MacroTransform with IdentityDenotTransformer  { thisTransformer =>
+class PostTyper extends MacroTransform with SymTransformer  { thisTransformer =>
+
 
   import tpd._
 
+  def transformSym(sym: SymDenotation)(implicit ctx: Context): SymDenotation = {
```

Comment:
Nitpick: `sym` is usually used for symbols, for a denotation I would use `ref` or `d`.